staging/src/k8s.io/apiserver/pkg/endpoints/filters/authorization.go

```diff
@@ -61,6 +61,9 @@ func WithAuthorization(handler http.Handler, a authorizer.Authorizer, s runtime.
 		if authorized == authorizer.DecisionAllow {
 			audit.LogAnnotation(ae, decisionAnnotationKey, decisionAllow)
 			audit.LogAnnotation(ae, reasonAnnotationKey, reason)
+			if err != nil {
+				glog.Warningf("authorization for %q passed but with an error: %v", req.RequestURI, err)
```

Comment:
is this actionable or especially concerning? if not, I'm not sure we should flood the logs with this.

Reply:
@liggitt I think we should at least output sth when a decision is make with an error. Or we won't have any idea of underlying errors from authn?

Reply:
you can also use `glog.V(N)....` here where `N` is a verbosity level.

Reply:
Interesting , how does authorization pass with errors :-) ? Just curious what errors are those ?